.env.dollar

```
@@ -21,3 +21,8 @@ NEXT_PUBLIC_THEME_SIDEBAR_ACCENT_FOREGROUND_LIGHT="145 70% 28%"
 NEXT_PUBLIC_THEME_SIDEBAR_ACCENT_LIGHT="142 60% 95%"
 NEXT_PUBLIC_THEME_STABLE_TOKEN="#85BB65"
 NEXT_PUBLIC_THEME_VOLATILE_TOKEN="#C026D3"
+
+NEXT_PUBLIC_NEUTRON_ID=TBD_DOLLAR_NEUTRON_ID
+NEXT_PUBLIC_PROTON_ID=TBD_DOLLAR_PROTON_ID
+NEXT_PUBLIC_GLUON_NFT=TBD_DOLLAR_GLUON_NFT
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_GLUON_NFT key should go before the NEXT_PUBLIC_NEUTRON_ID key

(UnorderedKey)